packages/sdk/src/middleware/middleware.ts

```diff
@@ -37,7 +38,16 @@ const actionToUri = (action: TSdkAction, projectKey?: string): string => {
   const service = requestBuilder[action.payload.service];
   if (action.payload.options) service.parse(action.payload.options);
 
-  return service.build();
+  return service.build({
+    // given `service=orderEdits` and given `applyOrderEditTo`, we build an apply endpoint
+    // given `service=orderEdits` and no `applyOrderEditTo`, we build an update endpoint
+    // https://docs.commercetools.com/api/projects/order-edits
+    applyOrderEdit:
+      action.payload.service === SUPPORTED_SERVICES.ORDER_EDITS &&
+      typeof action.payload.options?.applyOrderEditTo === 'string',
+    // at this stage, the `projectKey` should be available already.
+    withProjectKey: true,
```

Comment:
Why it was not there before? 🤔

Reply:
because it is defaulted https://github.com/commercetools/nodejs/blob/master/packages/api-request-builder/src/create-service.js#L128

I added it here to be more explicit.